network-api/networkapi/buyersguide/templates/primary_nav.html

```diff
@@ -30,3 +30,12 @@
     </div>
   </div>
 {% endblock %}
+
+{% block donate_and_newsletter %}
+<div class="d-flex align-items-center">
+  {% url 'why-view' as why_url %}
```

Comment:
just to keep things obvious, let's call these `about_...` instead of `why_...`

Reply:
the link goes to `/about/why` but i can update the variable to `about_why_url`